packages/inter-protocol/src/econCommitteeCharter.js

```diff
@@ -79,17 +79,50 @@ export const start = async zcf => {
     return zcf.makeInvitation(voteOnOfferFilterHandler, 'vote on offer filter');
   };
 
+  /**
+   * @param {Instance} instance
+   * @param {string} methodName
+   * @param {string[]} methodArgs
+   * @param {import('@agoric/time').TimestampValue} deadline
+   */
+  const makeApiInvocationInvitation = (
```

Comment:
There's no test for the charter?  Hmm. Seems like an oversight.

Reply:
I'm okay with it since we need to test uses of the charter to govern and we do in the `smartWallet` integration tests.